lib/private/help-exec-machine-instance.js

```diff
@@ -806,7 +806,7 @@ module.exports = function helpExecMachineInstance (liveMachine) {
       } catch (e) {
 
         // Split the error we got into lines, and remove the first one (the actual error message)
-        e.stack = e.stack.split('\n');
+        e.stack = stripStackMessage(e).split('\n');
```

Comment:
Can we use a local variable here instead of overwriting the Error instance's `.stack` property in place?


```js
var newStackEntries = getBareTraceFromError(e).split('\n');
```

_see also next comment_